api/test/HgResumeApi_Test.php

```diff
@@ -279,7 +279,7 @@ function testPullBundleChunk_PullUntilFinished_AssembledBundleIsValid() {
 
 		$assembledBundle = '';
 		$bundleSize = 1; // initialize the bundleSize; it will be overwritten after the first API call
-		while (mb_strlen($assembledBundle) < $bundleSize) {
+		while (mb_strlen($assembledBundle, "8bit") < $bundleSize) {
```

Comment:
I assume this is due to running this in a newer version of PHP, PHP 7.4?

Reply:
Honestely, I'm not quite sure. But almost all of the calls to `mb_strlen` **already** passed this as a paramter. And all the ones that **didn't** failed. So, it feels like this was an oversight.

Reply:
ok sounds good, thanks for the explanation.